packages/meditrak-app/app/assessment/specificQuestions/BinaryQuestion.js

```diff
@@ -4,7 +4,26 @@
  */
 
 import React from 'react';
-
+import PropTypes from 'prop-types';
 import { RadioQuestion } from './RadioQuestion';
 
-export const BinaryQuestion = props => <RadioQuestion options={['Yes', 'No']} {...props} />;
+export const BinaryQuestion = ({ options, ...props }) => {
+  const binaryOptions = options.length
```

Comment:
```suggestion
  const hasCustomOptions = !!options.length;
  const binaryOptions = hasCustomOptions
```
Could make things slightly more obvious